### ISSUE TITLE
Influx Slack Notification Tag support for `/` and `@`

### ISSUE THREAD

**Issue by Mikeburke14** [grafana/grafana#32493]
As discussed in #31284 

Tags formatted similar to `[[tag_/junos/rpm/probe-results/probe-test-results/@owner]]` do not render correctly within slack.

 `/` and `@` are valid and come from the [JTI OpenConfig Telemetry Input Plugin](https://github.com/influxdata/telegraf/blob/06c62f263c79b2ce04cf0d98841e58c51ea90ece/plugins/inputs/jti_openconfig_telemetry/README.md) which is part of the Influx GitHub

***Example Query***
![109141080-39c3f080-7755-11eb-9ab4-e105799d46bc](https://user-images.githubusercontent.com/140795/112985756-8174d800-9158-11eb-94a7-acd49eb10601.png)

***Example Slack message***
![109141102-40526800-7755-11eb-8d5b-17c998a5559a](https://user-images.githubusercontent.com/140795/112985768-86d22280-9158-11eb-94c4-4a6c1ebaba48.png)

**Environment**:
Grafana version: v7.5.0 (b144fb1a9f)
Data source type & version: Influx
OS Grafana is installed on: Debian GNU/Linux 10 (buster)
User OS & Browser: Chrome / Slack
Grafana plugins: Slack Notification

**Comment by stefi994** [comment]
Hey, @dsotirakis 
We upgraded our Grafana to v7.5.3 (3e3cf4dc60), unfortunately the issue is still there.
![image](https://user-images.githubusercontent.com/72752368/114175078-dcb58000-9939-11eb-8bbc-c30cb28856bf.png)

Can you please take another look? Thanks!

**Comment by dsotirakis** [comment]
@stefi994 yeah, didn't make it to `7.5.3` version - I'll make sure it'll make in on `7.5.4`!

Thanks for that!

**Comment by stefi994** [comment]
Hey @dsotirakis, I can't see this issue listed as fixed in the 7.5.4 release notes, can you please let us know if this issue is fixed in the 7.5.4 version?

Thanks!

**Comment by dsotirakis** [comment]
Hello @stefi994, it's there yeah, please give it a try and let us know! Thanks!

**Comment by stefi994** [comment]
Hey @dsotirakis,
Thanks! It's fixed now, the tags are correctly rendered :)
